packages/svelte/src/internal/client/reactivity/deriveds.js

```diff
@@ -94,6 +111,7 @@ export function update_derived(derived) {
 
 	if (DEV) {
 		let prev_inspect_effects = inspect_effects;
+		let prev_active_effect = active_effect;
```

Comment:
This is DEV only, you'll need to add it to the other path too?

Reply:
good catch! fixed